<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>591</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>591</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="welcome_widget" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 255);</string>
    </property>
    <widget class="QLabel" name="welcome_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Berlin Sans FB&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Welcome</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Berlin Sans FB&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Username : </string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>250</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>19</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 19pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127);
border-radius:20px</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 19pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127);
border-radius:20px</string>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Berlin Sans FB&quot;;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);
border-radius:20px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
